package.json

```
@@ -0,0 +1,35 @@
+{
+  "name": "@w3f/crypto",
+  "version": "0.0.1",
```

First version should be 0.1.0, according to semantic versioning, first feature added.
```suggestion
  "version": "0.1.0",
```